OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -585,12 +585,13 @@ class MoveIntoWorldActivity : Activity
 			WPos pos;
 			int delay;
 
-			public MoveIntoWorldActivity(Actor self, int delay = 0)
+			public MoveIntoWorldActivity(Actor self, int delay = 0, SubCell subCell = SubCell.Any)
```

Comment:
The subcell should not be set by the `MoveIntoWorldActivity`, but instead be added using a `SubCellInit` in `Production.DoProduction`.